### DISCUSSION TITLE
Homebox Home Project Planning

### DISCUSSION THREAD

**Discussion by katosdev** [sysadminsmedia/homebox#267] [Ideas]
Original suggestion posted on Reddit by u/Forsaken-Diamond-518

We have had the suggestion of including a home project tracker to track home improvement projects and their invoices, pictures around the project, linking assets to that project, etc.

Wanted to drop this discussion in to get community feedback on the idea.

**Comment by 101br03k** [comment]
Yeah most selfhosted project (management) solutions do not meet my requirements. 

I think this is way out of scope since it is apparently hard enough for people to a project management software when there only doing the project management software and not a inventory and organization system + a project management software.
